content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+main *Time Off* dashboard, or the :guilabel:`Create` button in the :guilabel:`My Allocations` list
+view. Both buttons open a new allocation request form. Enter the following information on the form:
+
+- :guilabel:`Allocation Name`: enter a name for the allocation. This should be descriptive such as
+  `Additional Sick Time 2024`.
+- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+  drop-down menu.
+- :guilabel:`Validity Period`: the current date populates the start date by default. If there is no
+  expiration on the time off type, there will be no date populated as the end date. If the time off
+  type has an expiration date, the date will automatically populate the end date field once the form
+  is saved.
+- :guilabel:`Duration`: enter the amount of time being requested. The format (either days or hours)
+  is in the same format as the time off type.
+- :guilabel:`Add a reason...`: enter a description for the allocation request. This should include
+  any details that the manager and approvers need in order to approve the request.
+
+When the form is complete, click the :guilabel:`Save` button to save the information and submit the
+request.
+
+.. image:: time_off/allocation-request.png
+   :align: center
+   :alt: An allocation request form filled out for an employee requesting an additional week of
+         sick time.
+
+Approvals
+=========
+
+Most requests for both time off and allocations need to go through the approval process prior to the
+time off being allocated and then granted to an employee. Requests either need one or two approvals,
+depending on how the specific type of time off type is configured.
+
+Only users who can approve allocation and time of requests will have the :guilabel:`Approvals`
+section visible in the *Time Off* application.
+
+Approve allocations
+-------------------
+
+To view allocations that need approval, navigate to :menuselection:`Time Off application -->
+Approvals --> Allocations`. The only allocations visible on this list are for employees the user
+has either :guilabel:`Time Off Officer` or :guilabel:`Administrator` access rights for the *Time
+Off* application.
+
+The default filters that are configured to be in place when navigating to the
+:guilabel:`Allocations` list are :guilabel:`My Team` and :guilabel:`Active Employee`. This presents
+only employees on the user's team (who they manage) and active employees. Inactive users are not
+shown.
+
+The left side of the screen has various grouping options to narrow down the presented allocation
+requests. The options are :guilabel:`To Approve`, :guilabel:`To Submit`, :guilabel:`Refused`, and
+:guilabel:`Approved`. To view all allocation requests, click :guilabel:`All`. It is also possible to
+display allocation requests by department. Click on the department to present only allocations for
+that specific department.
```

Comment:
Adding the list headings for clarity rather than only relying on stating it's location on the screen.
```suggestion
The left side of the screen has various grouping options to narrow down the presented allocation
requests by :guilabel:`STATUS`. The options are :guilabel:`To Approve`, :guilabel:`To Submit`,
:guilabel:`Refused`, and :guilabel:`Approved`. To view all allocation requests, click
:guilabel:`All`. It is also possible to display allocation requests by :guilabel:`DEPARTMENT`.
Click on the department to present only allocations for that specific department.
```